www/install/steps/process/createDbUser.php

```diff
@@ -97,11 +97,13 @@
 
     // checking mysql version before trying to alter the password plugin
     $prepareCheckVersion = $link->query($checkMysqlVersion);
+    $versionName = false;
+    $versionNumber = 0;
     while ($row = $prepareCheckVersion->fetch()) {
-        if (!isset($versionNumber) && $row['Variable_name'] === "version") {
-            $versionNumber = $row['version'];
-        } elseif (!isset($versionName) && $row['Variable_name'] === "version_comment") {
-            $versionName = $row['version_comment'];
+        if ($row['Variable_name'] === "version") {
+            $versionNumber = $row['Variable_name'];
+        } elseif ($row['Variable_name'] === "version_comment") {
+            $versionName = $row['Variable_name'];
         }
     }
     if ((strpos($versionName, "MariaDB") !== false && version_compare($versionNumber, '10.2.0') >= 0)
```

Comment:
```suggestion
    if ((strpos($versionName, "MariaDB") !== "" && version_compare($versionNumber, '10.2.0') >= 0)
```

Reply:
@callapa , is it the right path to take ?
If so, next line, needs to be modified too

Reply:
but I can't here... Waiting for your reply before going further